src/api/app/assets/javascripts/webui/packages.js

```diff
@@ -14,13 +14,13 @@ $(function ($) {
     }
   });
 
-  $('.expand-diffs').on('click', function () {
+  $('body').on('click', '.expand-diffs', function () {
```

Comment:
What makes this change necessary?

Reply:
It was because we re-render the partial in ajax and `$('.expand-diffs').on('click', function () {` was not enough to find the element in DOM.

Reply:
Could you add this to the commit description? This isn't obvious.

Reply:
Sure, I'll add it to the commit description.